spec/recipe_spec.rb

```diff
@@ -3,6 +3,7 @@
   ret = {'items' => 
```

Comment:
redundant info but it doesn't matter since this is fake data, only used to stub out api call while testing